README.md

```diff
@@ -1,2 +1,94 @@
-# Hello-Microverse
-This is the Hello Microverse Project from Week 1, Module 1
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Deployment](#deployment)
+- [👥 Author](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**Hello Microverse**  is a Microverse Project required in Module 1 of the Micorverse Full-Stack Program.
+
+
+
+<!-- GETTING STARTED -->
+
```

Comment:
- [ ] You did great using the provided Microverse readme template. :+1: :+1: Looking at the readme file, it is missing two important sections,  `Key Features` and `Future Features`. I'll suggest you return these sections and fill them in with the appropriate content, According to Microverse the only optional section is the FAQ, you are meant to have every other section in your readme file. :smile: :wave: